src/cpu/s390x/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+#===============================================================================
+# Copyright 2023 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+file(GLOB_RECURSE SOURCES
+    ${CMAKE_CURRENT_SOURCE_DIR}/*.[ch]pp
+    )
+
+set_source_files_properties(${SOURCES} PROPERTIES COMPILE_FLAGS "-mvx -mzvector")
+if(NOT UPPERCASE_CMAKE_BUILD_TYPE STREQUAL "DEBUG")
+set_source_files_properties(${SOURCES}
+    PROPERTIES COMPILE_FLAGS "-O3 -march=native -mtune=native -mvx -mzvector")
```

Comment:
are you sure about the use of `-march=native` ? This will typically force users to build oneDNN where it will run (and make distribution through packages not possible).

Reply:
right. I can omit that as there is ((CMAKE_SYSTEM_PROCESSOR STREQUAL CMAKE_HOST_SYSTEM_PROCESSOR))) check. The vectorization has been enabled since z13 arch.  should I add logic  to force the compiler to compile and tune for the specific architecture when the user specifies the target?